Tests/TestingApps/PaywallsTester/PaywallsTester/Views/OfferingsList.swift

```diff
@@ -78,7 +78,7 @@ struct OfferingsList: View {
             Text(error.description)
 
         case .none:
-            ProgressView()
+            SwiftUI.ProgressView()
```

Comment:
This wasn't compiling after #3526. I figured I'd fix it here instead of making another PR for one line.